File: LeetCode Solutions/May LeetCoding Challenges/Maximum Points You Can Obtain from Cards.cpp
class Solution {
public:
    int maxScore(vector<int>& cardPoints, int k) {
        int total = 0;
        for(int i=0; i<k; i++){
            total += cardPoints[i];
        }
        int maximumScore = total;
        for(int i=k-1, j=cardPoints.size()-1; i>=0; i--, j--){
            total += cardPoints[j] - cardPoints[i];
            maximumScore = max(maximumScore, total);
        }
        return maximumScore;
    }
};

File: LeetCode Solutions/May 2020 LeetCoding Challenge/Maximum Points You Can Obtain from Cards.py
class Solution:
    def maxScore(self, cardPoints: List[int], k: int) -> int:
        maximumScore = total = sum(cardPoints[:k])
        for i in range(k-1, -1, -1):
            total += cardPoints[i+len(cardPoints)-k] - cardPoints[i]
            maximumScore = max(maximumScore, total)
        return maximumScore